langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -113,12 +172,22 @@ public isolated function abs(float x) returns float = @java:Method {
 # (this is the round-to-nearest rounding mode, which is the default for IEEE and for Ballerina).
 # A value of `fractionDigits` greater than 0 thus corresponds to the number of digits after the decimal
 # point being `fractionDigits`; a value of 0 for `fractionDigits` rounds to an integer.
-# If `x` is NaN, +0, -0, +∞ or -∞, then the result is `x``.
+# If `x` is NaN, +0, -0, +∞ or -∞, then the result is `x`.
```

Comment:
Please send a PR fixing this to the spec repo also.

Reply:
Will update

Reply:
https://github.com/ballerina-platform/ballerina-spec/pull/1201